core/src/main/java/nav/statusplattform/core/repositories/RecordRepository.java

```diff
@@ -99,6 +96,34 @@ public List<RecordEntity> getRecordHistory(UUID serviceId, int maxNumberOfRecord
                 .list(RecordRepository::toRecord);
     }
 
+    public final List<RecordEntity> getRecordsInTimeSpan(UUID serviceId, ZonedDateTime from, ZonedDateTime to) {
+        System.out.println("getRecordsInTimeSpan");
+
```

Comment:
```suggestion
```

Reply:
Removed display line used as a part of testing from the code.